.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
-  "extends": "eslint-config-edx",
+  "extends": [
+    "eslint-config-edx",
+    "plugin:react-hooks/recommended"
```

Comment:
It's worth asking in FEDX if we would like to add this to our common eslint config: https://github.com/edx/eslint-config/blob/master/.eslintrc.js